Assets/_Scripts/3Managers/IslandManager.cs

```diff
@@ -11,6 +11,7 @@ public class IslandManager : Singleton<IslandManager>
 
     public Island Island;
     [SerializeField] IslandView view;
+    [SerializeField] Logger Logger;
```

Comment:
Can we make Logger static? Or maybe a singleton? I'd rather not initialize this through the inspector, especially if we want to use this in other managers as well.

Reply:
@JackRiga